### DISCUSSION TITLE
Broken Link in Wiki

### DISCUSSION THREAD

**Discussion by scottthealphanerd** [aqualinkd/AqualinkD#237] [General]
Obligatory awesome project kudos...  In the PDA section of the wiki there is a link to go to instructions on how to upgrade from PDA to RS.  The link target is "https://github.com/sfeakes/AqualinkD/wiki/Upgrading-Jandy-Aqualink-PDA-to-RS-panel" but appears broken and leads back to the wiki.

**Comment by sfeakes** [comment]
Sorry about that, I need to delete the link from the wiki.  Ultimately Jandy felt that explaining how to upgrade the PDA only panel to RS panel was infringing on their IP, so I had to delete the page.

**Reply by scottthealphanerd** [reply]
Interesting - was it the same commercial solution they have on their website or something more original?

**Reply by sfeakes** [reply]
I’m not sure what’s on their website.  You can basically change the chip on the board. The page simply described what chip you needed and what RS chip is compatible with what PDA version
